services/player-service/application/src/main/java/com/kjeldsen/player/application/usecases/GenerateSingleTrainingUseCase.java

```diff
@@ -27,26 +29,59 @@ public void generate(PlayerId playerId, List<PlayerSkill> skills, Integer days)
     }
 
     private void generateAndStoreEvent(PlayerId playerId, PlayerSkill playerSkill, int currentDay) {
-        PlayerTrainedEvent.builder()
+        PlayerTrainingEvent playerTrainingEvent = PlayerTrainingEvent.builder()
             .eventId(EventId.generate())
             .eventDate(Instant.now())
```

Comment:
Instant.now() -> InstantProvider

if the class does not exists in this project check match service I added it recently